packages/jest-cli/src/prepareStackTrace.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+// filter for noisy stack trace lines
+const FILTER_NOISE =
+  /(\/|\\)jest(-.*?)?(\/|\\)(vendor|build|node_modules|packages)(\/|\\)/;
+
+module.exports = (context: {Error: Class<Error>}) => {
+  const Error = context.Error;
+  Error.prepareStackTrace = (error: Error, stacktrace: CallSite[]): mixed => {
```

Comment:
Ah nice, I had a hunch that Flow knew about this and that the definitions are global but was hoping it not to be true because it is messy.